docs/docs-content/getting-started/aws/deploy-manage-k8s-cluster-tf.md

```diff
@@ -0,0 +1,748 @@
+token provided in the
+[Hello Universe](https://github.com/spectrocloud/hello-universe/tree/main?tab=readme-ov-file#reverse-proxy-with-kubernetes)
+repository for the authentication token.
+
+```hcl {7-8}
+##############################
+# Hello Universe Configuration
+##############################
+app_namespace   = "hello-universe" # The namespace in which the application will be deployed.
+app_port        = 8080             # The cluster port number on which the service will listen for incoming traffic.
+replicas_number = 1                # The number of pods to be created.
+db_password     = "REPLACE ME"     # The database password to connect to the API database.
+auth_token      = "REPLACE ME"     # The auth token for the API connection.
+```
+
+Locate the AWS provider section and change `deploy-aws = false` to `deploy-aws = true`. Additionally, replace all
+occurrences of `REPLACE_ME` with their corresponding values, such as those for the `aws-cloud-account-name`,
+`aws-region`, `aws-key-pair-name`, and `availability_zones` variables. You can also update the values for the nodes in
+the control plane or worker node pools as needed.
+
+:::warning
+
+Ensure that the SSH key pair specified in `aws-key-pair-name` is available in the same region specified by `aws-region`.
+For example, if `aws-region` is set to `us-east-1`, use the name of a key pair that exists in the `us-east-1` region.
+
+:::
+
+```hcl {4,7-9,16,24}
+###########################
+# AWS Deployment Settings
+###########################
+deploy-aws          = false # Set to true to deploy to AWS.
+deploy-aws-kubecost = false # Set to true to deploy to AWS and include Kubecost to your cluster profile.
+
+aws-cloud-account-name = "REPLACE ME"
+aws-region             = "REPLACE ME"
+aws-key-pair-name      = "REPLACE ME"
+
+aws_control_plane_nodes = {
+  count              = "1"
+  control_plane      = true
+  instance_type      = "m4.xlarge"
+  disk_size_gb       = "60"
+  availability_zones = ["REPLACE ME"] # If you want to deploy to multiple AZs, add them here. Example: ["us-east-1a", "us-east-1b"].
+}
+
+aws_worker_nodes = {
+  count              = "1"
+  control_plane      = false
+  instance_type      = "m4.xlarge"
+  disk_size_gb       = "60"
+  availability_zones = ["REPLACE ME"] # If you want to deploy to multiple AZs, add them here. Example: ["us-east-1a", "us-east-1b"].
+}
+```
+
+When you are done making the required changes, save the file.
+
+## Deploy the Cluster
+
+Before starting the cluster provisioning, export your [Palette API key](./setup.md#create-a-palette-api-key) as an
+environment variable. This step allows the Terraform code to authenticate with the Palette API.
+
+```bash
+export SPECTROCLOUD_APIKEY=<Your-Spectro-Cloud-API-key>
+```
+
+Next, issue the following command to initialize Terraform. The `init` command downloads the plugins and providers
```

Comment:
Which plugins? You never mentioned plugins until now?

Reply:
Good call! I was talking about the providers. Will change the sentence :)